core/XH.js

```diff
@@ -329,6 +329,17 @@ class XHClass {
         return this.acm.messageSourceModel.confirm(config);
     }
 
+    /**
+     * Show a modal 'prompt' dialog with a default TextInput, message and default 'OK'/'Cancel' buttons.
+     * Can provide a custom input control with the input prop (true to use default TextInput).
```

Comment:
The second sentence is confusing here -- need to reserve term 'prop' for react component prop, and try to use the term 'param' or 'option'.  Also, perhaps just 'Applications may also provide a custom HoistInput'.